config/schema/sessions.sql

```diff
@@ -5,9 +5,11 @@
 # Redistributions of files must retain the above copyright notice.
 # MIT License (https://opensource.org/licenses/mit-license.php)
 
-CREATE TABLE sessions (
-  id char(40) NOT NULL,
-  data text,
-  expires INT(11) NOT NULL,
-  PRIMARY KEY  (id)
-);
+CREATE TABLE `sessions` (
+  `id` char(40) CHARACTER SET ascii COLLATE ascii_bin NOT NULL,
+  `created` datetime DEFAULT CURRENT_TIMESTAMP,
+  `modified` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `data` blob DEFAULT NULL,
+  `expires` int(11) unsigned DEFAULT NULL,
```

Comment:
The digits limitation of the integer types in SQL are just for zero-filling, if that exists. However I will still change it, as it won't hurt and the current max unix timestamp is 32bit int (https://en.wikipedia.org/wiki/Unix_time)